src/components/RangeSlider.react.js

```diff
@@ -58,6 +58,17 @@ export default class RangeSlider extends Component {
             tipProps = tooltip;
         }
 
+        const omittedMarks = this.props.marks
+            ? Object.keys(this.props.marks).filter(value => {
+                  return value < this.props.min || value > this.props.max;
+              })
+            : [];
+
+        const truncatedMarks = omit(
+            omittedMarks.map(mark => parseInt(mark, 10)),
+            this.props.marks
+        );
```

Comment:
`parseInt` seems like a problem here - these might not be integers. But anyway looks like you can do this in one step using https://ramdajs.com/docs/#pickBy